modules/Dockerfile

```diff
@@ -15,33 +16,43 @@ FROM "${DOCKER_HUB_PROXY}python:3.9-slim-bullseye" as python-build
         libpoppler-cpp-dev \
         libfuzzy-dev \
         libssl-dev \
+        ninja-build \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     
     RUN mkdir /wheels
     WORKDIR /srv
 
-    RUN if [ ! -z ${MODULES_COMMIT} ]; then \
-        git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}; \
-        else git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules; fi
+    RUN <<-EOF
+            if [ ! -z ${MODULES_COMMIT} ]; then
+                git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}
+            else
+                git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules
+            fi
 
-    RUN cd /srv/misp-modules; \
-        echo "pyeti" >> REQUIREMENTS; \
-        echo "greynoise" >> REQUIREMENTS; \
-        echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS; \
-        sed -i 's|apiosintds.*|git+https://github.com/davidonzo/apiosintDS\@misp|g' REQUIREMENTS; \
-        sed -i 's/-e //g' REQUIREMENTS; \
-        pip3 wheel -r REQUIREMENTS --no-cache-dir -w /wheels/
+            cd /srv/misp-modules
+            echo "pyeti" >> REQUIREMENTS
+            echo "greynoise" >> REQUIREMENTS
+            echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS
+            sed -i 's|apiosintds.*|git+https://github.com/davidonzo/apiosintDS\@misp|g' REQUIREMENTS
```

Comment:
Looks like upstream's `misp-module` branch is now almost identical to the latest version AND pypi has an updated version.

In other words, we should probably use `apiosintds==2.0.1` rather than the git repo.

Reply:
Ah yeah, looks like the old sed was broken by that change too.

Reply:
Oh, how come? (I can't see the error)

Reply:
It doesn't match anything anymore because it's not case-insensitive (though that's a good thing in this case)

Reply:
Fetching the latest apiosintDS package is somewhat of a pain because it depends on validators==0.20.0, while misp-modules wants 0.14.0. I cannot find release notes for 0.14.0 to 0.20.0 (only 0.21 and upwards), so no easy way of knowing whether there are breaking changes in the move.

However, I cannot find any instance of the package actually being used in misp-modules; this applies to pyeti and greynoise too, so I'm thinking it's fine to just blindly go for the version apios wants.

Reply:
Updated the source now and let it choose validators release